app/views/annotation_categories/_uncategorized_annotations_list.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id='annotation_list_holder'>
+  <h2>
+    <%= t('annotations.non_categorized') %>
+  </h2>
+  <% if texts.empty? %>
+    <p>
+      <%= t('annotations.empty_uncategorized') %>
+    </p>
+  <% else %>
+    <% text_info =  AnnotationText.joins(annotations: { result: { grouping: :group } })
```

Comment:
This is a good idea, but you can improve this approach by merging this query with the one in the controller method. It's fine to make `@texts` itself an array of hashes rather than a collection of `AnnotationText` objects.